<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiblioJournalInterface</class>
 <widget class="QDialog" name="BiblioJournalInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiblioJournalInterface</string>
  </property>
  <widget class="QPushButton" name="confirmerButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameEntrees">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titreJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre de la reference: </string>
    </property>
   </widget>
   <widget class="QLabel" name="auteurJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Auteur: </string>
    </property>
   </widget>
   <widget class="QLabel" name="anneeJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Annee d'edition: </string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume: </string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero: </string>
    </property>
   </widget>
   <widget class="QLabel" name="pageJournal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTitreJournal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAuteurJournal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxAnneeJournal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2050</number>
    </property>
    <property name="value">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumero">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxVolume">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du journal: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNomJournal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="anneeJournal_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Code ISSN: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineIssnJournal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="annulerButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>annulerButton</sender>
   <signal>clicked()</signal>
   <receiver>BiblioJournalInterface</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmerButton</sender>
   <signal>clicked()</signal>
   <receiver>BiblioJournalInterface</receiver>
   <slot>validerEntrees()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerEntrees()</slot>
 </slots>
</ui>
